package.json

```diff
@@ -16,7 +16,7 @@
     "content-type"
   ],
   "dependencies": {
-    "mime-db": ">= 1.52.0 < 2"
+    "mime-db": ">= 1.54.0 < 2"
```

Comment:
Hm, yeah this is a fine update, but I am really wondering why not use `^1.54.0` here?

Reply:
oh!!! right this one was not in the upgrade path so didn't get a major. Cool, resolved.